MERN/Topic/Backend/Node/3.1_JSON.md

```diff
@@ -0,0 +1,152 @@
+## JSON 
+- It is a javascript object notation format very similar to xml.
+- Commonly used for APIs and Configs.
+- Lightweight and easy to read/write as compared to other formats like xml.
+- Integrates very nicely with almost all languages.
+
+
+### JSON types:
+
+- String   eg:"Hello"
+- Numbers  eg:10,-3,1.2e10
+- Boolean  eg:true,false
+- null  eg:null
+- Array  eg:[1,2,3],["hello","john"]
+- Object  eg:{"key":"value"},{"age":30}
+
+
+### How to create a JSON file:
+
+- Name anything you want,just end it with `.json`.eg:`companies.json`.
```

Comment:
Instead of end it, Please try to use something more technical like "give it extension of json".